pkg/network/tracer/process_cache.go

```diff
@@ -30,17 +30,17 @@ const (
 	maxProcessQueueLen = 100
 	// maxProcessListSize is the max size of a processList
 	maxProcessListSize     = 3
-	processCacheModuleName = "network_tracer__process_cache"
+	processCacheModuleName = "network_tracer.process_cache"
```

Comment:
prometheus metric names cannot contain `.`, which is why we use the double-underscore trick.